pkg/generators/models/testdata/cases/allof1/expected/model_foo.go

```diff
@@ -14,6 +18,11 @@ type Foo struct {
 	} `json:"bar,omitempty"`
 }
 
+// Validate implements basic validation for this model
+func (m Foo) Validate() error {
+	return validation.Errors{}.Filter()
```

Comment:
Is there an example where the embedded sub-type required validation?

Reply:
Maybe you can add it to the validation test schema, you can add an embedded address struct with street, city, country fields

Reply:
The basic subtype example covers this case. But you are right, I'll add it to the validation fixture

Reply:
@trusch i would like to see it with a more complex example that has validation on the subtype fields, i didn't notice that one when i when through the PR earlier

Reply:
I added `Address`